apps/experiments/models.py

```diff
@@ -433,19 +455,28 @@ class ExperimentRoute(BaseTeamModel):
     type = models.CharField(choices=ExperimentRouteType.choices, max_length=64, default=ExperimentRouteType.PROCESSOR)
     condition = models.CharField(max_length=64, blank=True)
 
+    parent_version = models.PositiveIntegerField(null=True, blank=True)
+    child_version = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
Are these necessary? If we make a copy of the route and update the `parent` and `child` reference to point to the versioned experiments then I don't think we need these.